doc/man1/flux-job.rst

```diff
@@ -0,0 +1,97 @@
+.. flux-help-include: true
+
+===========
+flux-job(1)
+===========
+
+
+SYNOPSIS
+========
+
+**flux** **job** **cancel** *id* [*message...*]
+
+**flux** **job** **cancelall** [*OPTIONS*] [*message...*]
+
+**flux** **job** **kill** [*--signal=SIG*] *id*
+
+**flux** **job** **killall** [*OPTIONS*]
+
+**flux** **job** **raise** [*OPTIONS*] *id* [*message...*]
+
+**flux** **job** **raiseall** [*OPTIONS*] *type* [*message...*]
+
+DESCRIPTION
+===========
+
+flux-job(1) performs various job related housekeeping functions.
+
+CANCEL
+======
+
+A single job may be canceled with ``flux job cancel``.
+
+Jobs may be canceled in bulk with ``flux job cancelall``.  Target jobs are
+selected with:
+
+**-u, --user=USER**
+   Set target user.  The instance owner may specify *all* for all users.
+
+**-S, --states=STATES**
+   Set target job states (default: ACTIVE).
+
+**-f, --force**
+   Confirm the command
+
+**-q, --quiet**
+   Suppress output if no jobs match
+
+SIGNAL
+======
+
+Running jobs may be signaled with ``flux job kill``.
+
+**-s, --signal=SIG**
+   Send signal SIG (default: SIGTERM).
+
+Running jobs may be signaled in bulk with ``flux job killall``.  In addition
+to the option above, target jobs are selected with:
+
+**-u, --user=USER**
+   Set target user.  The instance owner may specify *all* for all users.
+
+**-f, --force**
+   Confirm the command.
+
+EXCEPTION
+=========
+
+An exception may raised on a single job with ``flux job raise``.
+
+**-s, --severity=N**
+   Set exception severity.  The severity may range from 0=fatal to
+   7=least severe (default: 0).
+
+**-t, --type=TYPE**
+   Set exception type (default: cancel).
+
+Exceptions may be raised in bulk with ``flux job raiseall``, which requires a
+type (positional argument) and accepts the following options:
```

Comment:
not for this issue, but the fact `raiseall` takes a positional `type` but `raise` takes it as an option seems sort of weird. perhaps an issue to be fixed?

Reply:
I actually did change that, but while working on tests realized that the default severity is 7, so `cancel` would not be a reasonable default type.  What would be a reasonable default to go with 7?  We don't define any standard, non-fatal exception types.  If type is required, then the tiny @grondo in my brain says it should not be an option but a required positional argument.  We could change the default severity to 0 and go with a default type of `cancel`, but that's what `cancelall` is for.  So meh, I reverted the change and went with the status quo.

Reply:
ahh ok, makes sense.  just seemed weird at a glance.

Reply:
Yeah, to me too.